### ISSUE TITLE
Error reporting sequences

### ISSUE THREAD

**Issue by mrpanda** [YugaByte/yugabyte-db#1666]
I was able to create a sequence successfully.
However, I have not been able to list it in the catalog. This may be related to other catalog synchronization issues that are already known, so sorry if this is a repeat.

Here is a copy of what I did and the output received:
```
postgres=# \c pbl_play
You are now connected to database "pbl_play" as user "postgres".
pbl_play=# select * from information_schema.sequences;
select * from information_schema.sequences;
ERROR:  Network error: GetTableLocations(pbl_play.pg_depend, , 1) failed: recvmsg error: Connection reset by peer (error 104)
```
If I restart the cluster everything appears to be okay. So I tried again and this is the result I received:
```
pbl_play=# select * from information_schema.sequences;
select * from information_schema.sequences;
ERROR:  Timed out: Read(tablet: 00000000000000000000000000000000, num_ops: 1, num_attempts: 343, txn: 00000000-0000-0000-0000-000000000000) passed its deadline 908.109s (passed: 59.998s): Network error (yb/util/net/socket.cc:590): recvmsg error: Connection reset by peer (error 104)
pbl_play=# 
```
Which appears to be a timeout, possibly related to the first problem locking a resource that is no longer available.

**Comment by hectorgcr** [comment]
@mrpanda Are you still able to reproduce this issue? I was unable to reproduce it locally.

**Comment by mrpanda** [comment]
Hi Hector,

yes, I refreshed my play database and re-ran the select statement and this time I received a timeout error - 

postgres=# \c pbl_play
You are now connected to database "pbl_play" as user "postgres".
pbl_play=# select * from information_schema.sequences;
select * from information_schema.sequences;
ERROR:  Timed out: Read(tablet: 00000000000000000000000000000000, num_ops: 1, num_attempts: 343, txn: 00000000-0000-0000-0000-000000000000) passed its deadline 190.789s (passed: 60.072s): Network error (yb/util/net/socket.cc:590): recvmsg error: Connection reset by peer (error 104)

Note that there are 4 sequences created in pbl_play. Maybe it is susceptible to slow performance ?

**Comment by mrpanda** [comment]
Hi Hector,

I allocated more memory to my VM and rebooted and did the test again - I don't receive performance problems when I do other queries - just this. This time I got the original error --

postgres=# \c pbl_play
You are now connected to database "pbl_play" as user "postgres".
pbl_play=# select max(id) from n1tbl1;
select max(id) from n1tbl1;
  max  
-------
 19701
(1 row)

pbl_play=# select count(*) from information_schema.sequences;
select count(*) from information_schema.sequences;
ERROR:  Network error: GetTableLocations(pbl_play.pg_depend, , 1) failed: recvmsg error: Connection reset by peer (error 104)
pbl_play=#

**Comment by hectorgcr** [comment]
Thanks @mrpanda. I will try to reproduce it. It seems to be an issue with the master timing out and not directly related to sequences. Would it be possible to send me the logs from the master leader?

**Comment by mrpanda** [comment]
Hector, could you specify the dir and filename you are looking for. I looked in yugabyte-data directory and there are lots of different log files.

**Comment by hectorgcr** [comment]
Stephen, you can run this command to generate a file with all the masters log files:
`find $(bin/yb-ctl status | grep "Cluster Data" | awk '{print $5}') -type f  -a \( -name "*master*.INFO*" -o -name "*master*.ERROR*" -o -name "*master*.WARNING*" -o -name "*master*.FATAL*" -o -name "*master*.err" -o -name "*master*.out" \) -print0 | tar -cvzf yugabyte_logs.tgz --null -T -`

You will have to run it in the yugabyte directory because you will need access to `bin/yb-ctl` to determine the location of the `yugabyte-data` directory.

**Comment by mrpanda** [comment]
Hi Hector,

I have attached the file as a result of your command. Note that Github does not accept tgz so i appended a gz to the file name. Thanks.

[yugabyte_logs.tgz.gz](https://github.com/YugaByte/yugabyte-db/files/3356867/yugabyte_logs.tgz.gz)

**Comment by hectorgcr** [comment]
Thanks for sending the logs Stephen. We found a couple of crashes in the master:

```
F20190701 20:33:27 ../../src/yb/docdb/ql_rocksdb_storage.cc:129] Check failed: !request.has_paging_state() Optimization failure due to wrong assumption
    @     0x7ffa8aef44b5  yb::LogFatalHandlerSink::send()
    @     0x7ffa89cd4666  google::LogMessage::SendToLog()
    @     0x7ffa89cd1d5a  google::LogMessage::Flush()
    @     0x7ffa89cd4c89  google::LogMessageFatal::~LogMessageFatal()
    @     0x7ffa923dc233  yb::docdb::QLRocksDBStorage::GetIterator()
    @     0x7ffa923d4ac6  yb::docdb::PgsqlReadOperation::Execute()
    @     0x7ffa926c660b  yb::tablet::AbstractTablet::HandlePgsqlReadRequest()
    @     0x7ffa926cd20d  yb::tablet::Tablet::HandlePgsqlReadRequest()
    @     0x7ffa92a5fe6c  yb::tserver::TabletServiceImpl::DoRead()
    @     0x7ffa92a611c1  yb::tserver::TabletServiceImpl::CompleteRead()
    @     0x7ffa92a628ef  yb::tserver::TabletServiceImpl::Read()
    @     0x7ffa8f2e718a  yb::tserver::TabletServerServiceIf::Handle()
    @     0x7ffa8e0718e8  yb::rpc::ServicePoolImpl::Handle()
    @     0x7ffa8e01f3c4  yb::rpc::InboundCall::InboundCallTask::Run()
    @     0x7ffa8e07cfd8  yb::rpc::(anonymous namespace)::Worker::Execute()
    @     0x7ffa8af6cab9  yb::Thread::SuperviseThread()
    @     0x7ffa86768694  start_thread
    @     0x7ffa85ea541d  __clone
    @              (nil)  (unknown)
```
and 
```
F20190628 20:41:31 ../../src/yb/master/catalog_manager.cc:3234] Check failed: 'ns.get()' Must be non NULL 
    @     0x7f744d4ff4b5  yb::LogFatalHandlerSink::send()
    @     0x7f744c2df666  google::LogMessage::SendToLog()
    @     0x7f744c2dcd5a  google::LogMessage::Flush()
    @     0x7f744c2dfc89  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f74553fdb9b  yb::master::CatalogManager::ListTables()
    @     0x7f74553e63aa  yb::TablesCollector::Collect()
    @     0x7f74553daa7c  yb::CollectorBase::Run()
    @     0x7f74553db4f9  yb::CallHome::BuildJson()
    @     0x7f74553dbccd  yb::CallHome::BuildJsonAndSend()
    @     0x7f74553dbd4c  yb::CallHome::DoCallHome()
    @     0x7f745067359a  _ZN5boost4asio6detail18completion_handlerIZN2yb3rpc9Scheduler4Impl11HandleTimerERKNS_6system10error_codeEEUlvE_E11do_completeEPvPNS1_19scheduler_operationESA_m
    @     0x7f745062f0b4  yb::rpc::IoThreadPool::Impl::Execute()
    @     0x7f744d577ab9  yb::Thread::SuperviseThread()
    @     0x7f7448d73694  start_thread
    @     0x7f74484b041d  __clone
    @              (nil)  (unknown)
```

The first issue has already been fixed, but it hasn't been released yet. We are hoping to have a release by Friday. I will let you know as soon as it happens.

Also, the first crash is what is causing the error that you are seeing when querying the information_schema table. In the meantime, you can run this command
```
bin/yb-ctl restart --tserver_flags="ysql_prefetch_limit=4096"
```
It should fix your issue.

**Comment by hectorgcr** [comment]
Stephen, we have a new release with the fix.

**Comment by mrpanda** [comment]
Hi Hector,

your workaround resolves the problem. I will download the latest version and re-test. Thanks.

**Comment by mrpanda** [comment]
Hector,

I was able to confirm that this is fixed in 1.2.12.0  Thanks
